package/kernel/linux/modules/usb.mk

```diff
@@ -1694,7 +1694,8 @@ define KernelPackage/usb3
 	+TARGET_bcm53xx:kmod-usb-bcma \
 	+TARGET_bcm53xx:kmod-phy-bcm-ns-usb3 \
 	+TARGET_ramips_mt7621:kmod-usb-xhci-mtk \
-	+(TARGET_apm821xx_nand&&LINUX_5_10):kmod-usb-xhci-pci-renesas
+	+(TARGET_apm821xx_nand&&LINUX_5_10):kmod-usb-xhci-pci-renesas \
+	+(TARGET_mvebu_cortexa9&&LINUX_5_10):kmod-usb-xhci-pci-renesas
```

Comment:
mvebu (and apm821xx) are now both on `KERNEL_PATCHVER:=5.10`. I guess the dependency on the kernel can be dropped now.

Reply:
done.